api/src/main/java/uk/gov/dhsc/htbhf/card/controller/CardController.java

```diff
@@ -32,10 +32,10 @@
 public class CardController {
 
     @PostMapping
-    @ApiOperation("Create a new card")
+    @ApiOperation("Create a new card request")
```

Comment:
I don't think we need to add request here, to me that sounds like we're getting back a request and are not creating a card.

Reply:
Yes, good spot, I was a bit over enthusiastic with that one.